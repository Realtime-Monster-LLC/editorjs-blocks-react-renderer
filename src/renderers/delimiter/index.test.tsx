import React from 'react';
import { create } from 'react-test-renderer';
import Delimiter from '.';

describe('<Delimiter />', () => {
  describe('when receives a Delimiter block', () => {
    it('renders a <hr /> tag', () => {
      expect(create(<Delimiter className="paragraph-delimiter" />).toJSON()).toMatchSnapshot();
    });
  });
});
