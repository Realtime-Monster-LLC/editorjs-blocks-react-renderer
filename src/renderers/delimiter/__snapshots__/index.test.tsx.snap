// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Delimiter /> when receives a Delimiter block renders a <hr /> tag 1`] = `
<hr
  className="paragraph-delimiter"
/>
`;
